### PR TITLE
connect to search api

### PR DESCRIPTION
**Pull request by rantifamilusi**
implement search api

### CODE REVIEW

File: app/JobManager/API/Models/PeopleSearchResponse.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace JobManager.API.Models
+{
+    public class PeopleSearchResponse
```
Comment (alexjoybc) [inline app/JobManager/API/Models/PeopleSearchResponse.cs]: Should this be moved to SearchApi.core so both api and jobmanager use the same object?

File: app/JobManager/API/Helpers/HttpClient.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        public virtual async Task<T> Get<T>(string baseUrl, string path, string key = null,object queryParams = null, object headers = null, object cookies = null)
+        {
+            return await new Url(baseUrl)
+                .AppendPathSegment(path)
+                .SetQueryParams(queryParams ?? new { })
+                .WithClient(_client)
+                .WithTimeout(HttpRequestTimeout)
+                .WithCookies(cookies ?? new { })
```
Comment (alexjoybc) [inline app/JobManager/API/Helpers/HttpClient.cs]: Are we planning on using cookies?

File: app/JobManager/API/Models/PeopleSearchResponse.cs (hunk 1 above)
Comment (rantifamilusi) [reply app/JobManager/API/Models/PeopleSearchResponse.cs]: I would rather we move it to DataModel, it's why i created the project and we can add to search pi core and jobmanager and anywhere we may need it

File: app/JobManager/API/Helpers/HttpClient.cs (hunk 2 above)
Comment (rantifamilusi) [reply app/JobManager/API/Helpers/HttpClient.cs]: Well.. it's a generic implementation that can be used to connect.. I think we should leave it in

File: app/JobManager/API/Models/PeopleSearchResponse.cs (hunk 1 above)
Comment (rantifamilusi) [reply app/JobManager/API/Models/PeopleSearchResponse.cs]: pushed changes.. moved to Datamodel

Comment (alexjoybc) [reply app/JobManager/API/Models/PeopleSearchResponse.cs]: DataModel and core are kind of competing, we should bring them together...

Comment (rantifamilusi) [reply app/JobManager/API/Models/PeopleSearchResponse.cs]: Oh okay.. let talk about it after the meeting and decide. I can make those changes

**Pull request merged**